openpype/settings/entities/schemas/projects_schema/schema_project_global.json

```diff
@@ -5,6 +5,61 @@
     "label": "Global",
     "is_file": true,
     "children": [
+        {
+            "type": "dict",
+            "key": "version_start_category",
+            "label": "Version Start",
+            "collapsible": true,
+            "collapsible_key": true,
+            "children": [
+                {
+                  "type": "list",
+                  "collapsible": true,
+                  "key": "profiles",
+                  "label": "Profiles",
+                  "object_type": {
+                      "type": "dict",
+                      "children": [
+                          {
+                              "key": "hosts",
```

Comment:
```suggestion
                              "key": "host_names",
```

Reply:
Is this because you want to make a distinction between a `host` object and the name of a host?